### ISSUE TITLE
Update Packet Generator provisioning code to support nfvbench 2.0.3, multi-chaining tests and reserved/on demand instances

### ISSUE THREAD

**Issue by lixuna** [cncf/cnfs#116]
Ansible tasks for - _Update Ansible playbooks to match expected environment and configuration_:
- [x] Create and test ansible playbook for intel
  - [x] Configure interfaces to match configuration
    * (X710 4x10GbE) 1) Bond0, 2) VLAN_1, 3) VLAN_2, 4) Unused
  - [x] Add additional steps for using Intel NIC
    * Load vfio-pci kernel module at boot
  - [x] Update NFVbench role to use NFVbench 2.x
    * _details TBD_

- [x] update the packet_generator deploy script to support provisioning existing nodes.

---

Terraform tasks:

- [x] update deploy scripts to use different TF override files
- [x] create a TF override file for the dual port that's empty
- [x] create a TF override file for the quad port intel that uses reserved instances

---

- [ ] Integrate packet generator updates from #92 as of Nov 7th, 2018
   *details TBD*

**Comment by taylor** [comment]
@pmikus, if your available help with integrating the packet generator changes from #92 into the oprovisioning code  ([ansible](https://github.com/cncf/cnfs/tree/master/comparison/ansible) and [deploy config](https://github.com/cncf/cnfs/tree/master/comparison/chained_nf_test/packet_generator)) would be appreciated.

The Ansible code can be tested w/o the terraform or together.

**Comment by pmikus** [comment]
> @pmikus, if your available help with integrating the packet generator changes from #92 into the oprovisioning code ([ansible](https://github.com/cncf/cnfs/tree/master/comparison/ansible) and [deploy config](https://github.com/cncf/cnfs/tree/master/comparison/chained_nf_test/packet_generator)) would be appreciated.
> 
> The Ansible code can be tested w/o the terraform or together.

With my pleasure. Looking on it.

**Comment by lixuna** [comment]
Update: 

Peter Mikus [8 hours ago]
>>> After testing and bug fixing I can confirm that 2.0.3 is the version that works smoothly now.

**Comment by pmikus** [comment]
I update all instances of packet generator code.
We may want to override VLAN setting of nfvbench to match destination. But other then that, done,

**Comment by taylor** [comment]
@pmikus, we can add the VLAN setting override in another issue after #113 is done.
